src/docfx/restore/FileResolver.cs

```diff
@@ -63,6 +63,20 @@ public string ReadString(SourceInfo<string> file)
             return reader.ReadToEnd();
         }
 
+        public byte[] ReadAllBytes(SourceInfo<string> file)
```

Comment:
nit: `ReadBytes` to align with other methods like `ReadString`
```suggestion
        public byte[] ReadBytes(SourceInfo<string> file)
```